sigle/src/modules/layout/components/DashboardLayout.tsx

```diff
@@ -12,6 +12,40 @@ import {
   AccordionTrigger,
 } from '../../../ui/Accordion';
 import { isExperimentalAnalyticsPageEnabled } from '../../../utils/featureFlags';
+import { VariantProps } from '@stitches/react';
+
+const DashboardContainer = styled(Container, {
+  flex: 1,
+  mt: '$10',
+  width: '100%',
+  display: 'grid',
+
+  '@md': {
+    justifyContent: 'center',
+    gridTemplateColumns: '724px',
+  },
+
+  '@lg': {
+    gridTemplateColumns: '834px',
+  },
+
+  variants: {
+    layout: {
+      default: {
+        '@xl': {
+          gridTemplateColumns: '1fr 826px 1fr',
+        },
+      },
+      wide: {
+        gridTemplateColumns: '1fr 5fr',
```

Comment:
working solution to get analytics content to fill the rest of the available space, whilst keeping the same relative distance to the sidebar as on other pages. Just wondering if there may be a better way to achieve this?

Reply:
Looks like it's working fine that way 👍